controllers/users.js

```diff
@@ -16,7 +16,11 @@ const secret = process.env.SECRET;
 const signup = async (req, res, next) => {
   const { error } = userValidator(req.body);
   if (error) return res.status(400).json({ message: error.details[0].message });
+  // Logowanie przekazywanych danych
+  console.log("Received email:", req.body.email);
```

Comment:
Przed zmergowaniem kodu do brancha master warto zadbać o to, aby usunąć wszelkie niepotrzebne console.log oraz zbędne komentarze. To pomaga w utrzymaniu czystości i profesjonalizmu kodu, co jest szczególnie ważne w produkcji. Czysty kod jest łatwiejszy w utrzymaniu i zrozumieniu dla innych deweloperów.